deploy/opt/bfi/iiif-logging/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.9"
+
+services:
+  app:
+    image: ghcr.io/bfidatadigipres/bfi-iiif-logging:${LOGGING_IMAGE_TAG}
+    environment:
+      - MYSQL_HOST=mysql:3306
+      - MYSQL_USERNAME=logging
+      - MYSQL_PASSWORD_FILE=/run/secrets/passwords/mysql # Double underscore is intentional.
```

Comment:
I think this is a copy/paste error - the double underscore only exists on the Nginx Proxy Manager container, for reasons unknown to me. MySQL uses a single underscore.

Reply:
Oops, I deleted the underscore but left the comment.